src/Resources/MaterialRequirements/MrDetailResource.cs

```diff
@@ -37,5 +37,31 @@ public class MrDetailResource
         public string Week12 { get; set; }
         
         public MrDetailItemResource ImmediateItem { get; set; }
+
```

Comment:
why are there only 12 weeks - is that just for during development and they'll be more later or some other reason.

Reply:
each block shows relative weeks 0 -12 so it's a quarter basically. They can scroll through quarter by quarter or show them all at once as they wish.